.github/workflows/build.yml

```
@@ -30,9 +30,17 @@ jobs:
         with:
           board_id: ${{ matrix.board.id }}
           arch: ${{ matrix.board.arch }}
+  BuildSdkPosix:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v6
+        with:
+          persist-credentials: false
+      - name: "Build SDK"
+        uses: ./.github/actions/build-sdk-posix
```

actionlint (1.7.12)

[error] description is required in metadata of "Build" action at "/home/jailuser/git/.github/actions/build-sdk-posix/action.yml"

(action)